beetmoverscript/templates/firefox_nightly_en_us_signed.yml

```diff
@@ -5,20 +5,16 @@ mapping:
 {% for locale in ['en-US'] %}
   {{ locale }}:
   {% if platform in ["linux-i686", "linux-x86_64"] %}
-    package:
-      artifact: target.tar.bz2
+    target.tar.bz2:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.tar.bz2
-    complete_mar:
-      artifact: update/target.complete.mar
+    target.complete.mar:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.complete.mar
       update_balrog_manifest: true
-    checksums:
-      artifact: target.checksums
-      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums
-    checksums_asc:
-      artifact: target.checksums.asc
-      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums.asc
-
+# Bug 1316124 - figure out checksums
```

Comment:
like the other places, I'll leave all signing task based checksum files out of the manifest until we discuss